extension-androidauto/src/main/java/com/mapbox/maps/extension/androidauto/LogoWidget.kt

```diff
@@ -0,0 +1,28 @@
+package com.mapbox.maps.extension.androidauto
+
+import android.content.Context
+import android.graphics.BitmapFactory
+import com.mapbox.maps.renderer.widget.BitmapWidget
+import com.mapbox.maps.renderer.widget.WidgetPosition
+
+/**
+ * Widget shows compass. Positioned in the bottom left corner by default.
+ *
+ * @param position position of logo
+ * @param marginX horizontal margin in pixels
+ * @param marginY vertical margin in pixels
+ */
+class LogoWidget(
```

Comment:
missing JvmOverloads and uses params with default values

Reply:
Discussed - leaving as is for the moment, overloaded constructors does not look very nice in Java neither